src/components/List/List.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const List = ({ lists }) => {
+  const listElements = lists.map(list => (
```

Comment:
items.map(item => ....

that sounds better i think

Reply:
Thanks for the suggestions. I cleaned up the code.